src/components/BookItem.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { deleteBook } from '../redux/books/booksSlice';
+/*eslint-disable*/
```

Comment:
- [ ] Please make sure that never disable the eslinter in your code, try to understand why are you getting the eslinter for that specific line or file. Because it is also against the general requirements of the project. Kindly remove it and try to resolve the issue without disabling linters.